controllers/user.js

```diff
@@ -234,8 +234,29 @@ exports.userListPage = function (aReq, aRes, aNext) {
       pageMetadata(options, ['Flagged Users', 'Moderation']);
     }
   }
-  function render() { aRes.render('pages/userListPage', options); }
-  function asyncComplete(err) { if (err) { return aNext(); } else { preRender(); render(); } }
+  function render() {
+    aRes.render('pages/userListPage', options);
+  }
+  function asyncComplete(aErr) {
+    if (aErr) {
+      aNext();
+      return;
+    }
```

Comment:
asyncComplete should always test for an error and handle it (at the very least a 500 page). If we can catch an error earlier on then great, but this is really the last line of defense. It would be nice if we could find a way to abstract this function and put it in libs/helpers.js since I hate seeing the same logic repeated, but I realize the difficulty of doing this.